ChangeLog.d/bugfix_PR3802.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix
+   * Fix build warning using cmake 3.19.0-rc1. Cmake soon drops support for
+     cmake_minimum_required(VERSION 2.6). This is now upgraded to require
+     cmake 2.8.12. See issue #3801
```

Comment:
This is not really a bug fix: CMake 3.19.0 isn't even released yet, and even if it will probably be by the time people read this, it's only a warning. On the other hand, this is a requirement change. So it should be announced under “Requirement changes”, and not under “Bugfix”.

Suggestion (to be wrapped to 80 columns):

> Requirement changes
>    * Update the minimum required CMake version to 2.8.12. This silences a warning on CMake 3.19.0. #3801

Reply:
@gilles-peskine-arm makes good sense. I have also renamed 
 ChangeLog.d/bugfix_PR3802.txt -> ChangeLog.d/minimum_cmake_version_PR3802.txt